nb.rs,"HTTPSConnectionPool(host='nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.nb.rs', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))"
nbuv.gov.ua,"HTTPSConnectionPool(host='nbuv.gov.ua', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa92526c190>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='nbuv.gov.ua', port=80): Read timed out. (read timeout=2)"
ncc.gov.ng,"HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ncc.gov.ng', port=443): Read timed out. (read timeout=2)"
neik.vn,"HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neik.vn', port=443): Read timed out. (read timeout=2)"
neliti.com,"HTTPSConnectionPool(host='neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.neliti.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
neopod.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","HTTPConnectionPool(host='neopod.ru', port=80): Read timed out. (read timeout=2)"
neowing.co.jp,"HTTPSConnectionPool(host='neowing.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa927274040>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='neowing.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa924b94d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
net.ee,"HTTPSConnectionPool(host='net.ee', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa922ab0370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='net.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa924a7dcd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
netky.sk,"HTTPSConnectionPool(host='netky.sk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa924436b20>, 'Connection to netky.sk timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.netky.sk', port=80): Read timed out. (read timeout=2)"
neturuguay.com,"HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='neturuguay.com', port=443): Read timed out. (read timeout=2)"
netwerkgalerij.be,"HTTPSConnectionPool(host='netwerkgalerij.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa921dd98e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='netwerkgalerij.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa921f5a0a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newedc.com,"HTTPSConnectionPool(host='www.newedc.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.newedc.com', port=443): Read timed out. (read timeout=2)"
newellwind.com,"HTTPSConnectionPool(host='newellwind.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa921ac4fa0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='newellwind.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa925baf6d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
newpages.com.my,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","HTTPSConnectionPool(host='www.newpages.com.my', port=443): Read timed out. (read timeout=2)"
news-mail.com.au,"HTTPSConnectionPool(host='www.couriermail.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.couriermail.com.au', port=443): Read timed out. (read timeout=2)"
newsformy.com,"HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='newsformy.com', port=443): Read timed out. (read timeout=2)"
nfe.go.th,"HTTPSConnectionPool(host='nfe.go.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='nfe.go.th', port=80): Read timed out. (read timeout=2)"
nguoiduatin.vn,"HTTPSConnectionPool(host='nguoiduatin.vn', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa91f1d1040>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nguoiduatin.vn', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa926102070>, 'Connection to nguoiduatin.vn timed out. (connect timeout=2)'))"
nhk.or.jp,"HTTPSConnectionPool(host='nhk.or.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa9277cecd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nhk.or.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91f1b9ac0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ni.rs,"HTTPSConnectionPool(host='ni.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa91ea9c730>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ni.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91ed0e880>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nihon-kankou.or.jp,"HTTPSConnectionPool(host='nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nihon-kankou.or.jp', port=443): Read timed out. (read timeout=2)"
nii.ac.jp,"HTTPSConnectionPool(host='nii.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa927209250>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nii.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa92741a1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
niigata.lg.jp,"HTTPSConnectionPool(host='niigata.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa920b31340>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='niigata.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa9204dfdf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nikkei.co.jp,"HTTPSConnectionPool(host='nikkei.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa92611f100>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nikkei.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa92668c1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nilephotos.net,"HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nilephotos.net', port=443): Read timed out. (read timeout=2)"
ninhthuan.gov.vn,"HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.ninhthuan.gov.vn', port=80): Read timed out. (read timeout=2)"
nippon.com,"HTTPSConnectionPool(host='nippon.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa925018d00>, 'Connection to nippon.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nippon.com', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa920c950d0>, 'Connection to nippon.com timed out. (connect timeout=2)'))"
nissen.co.jp,"HTTPSConnectionPool(host='nissen.co.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'nissen.co.jp' doesn't match either of '*.azurewebsites.net', '*.scm.azurewebsites.net', '*.azure-mobile.net', '*.scm.azure-mobile.net', '*.sso.azurewebsites.net'"")))","HTTPSConnectionPool(host='www.nissen.co.jp', port=443): Read timed out. (read timeout=2)"
nitra.sk,"HTTPSConnectionPool(host='nitra.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'nitra.sk' doesn't match either of '*.koronavirus.nitra.sk', 'koronavirus.nitra.sk'"")))","HTTPSConnectionPool(host='www.nitra.sk', port=443): Read timed out. (read timeout=2)"
njuzz.ee,"HTTPSConnectionPool(host='njuzz.ee', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa92577e1f0>, 'Connection to njuzz.ee timed out. (connect timeout=2)'))","HTTPConnectionPool(host='njuzz.ee', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa926dec7f0>: Failed to establish a new connection: [Errno 113] No route to host'))"
nms-laboratorija.lv,"HTTPSConnectionPool(host='nms-laboratorija.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='nms-laboratorija.lv', port=80): Read timed out. (read timeout=2)"
nojima.co.jp,"HTTPSConnectionPool(host='nojima.co.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa9267e8f10>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nojima.co.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa925d009d0>, 'Connection to nojima.co.jp timed out. (connect timeout=2)'))"
nomos-elibrary.de,"HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nomos-elibrary.de', port=443): Read timed out. (read timeout=2)"
noorageagea.net,"HTTPSConnectionPool(host='noorageagea.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa926512eb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='noorageagea.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91f149100>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
norham.org,"HTTPSConnectionPool(host='norham.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='norham.org', port=80): Read timed out. (read timeout=2)"
norrisnwhomes.com,"HTTPSConnectionPool(host='norrisnwhomes.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa920ca1c10>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='norrisnwhomes.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa921c3dd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
northernstar.com.au,"HTTPSConnectionPool(host='www.dailytelegraph.com.au', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.dailytelegraph.com.au', port=443): Read timed out. (read timeout=2)"
novine.hr,"HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novine.hr', port=443): Read timed out. (read timeout=2)"
noviteroditeli.bg,"HTTPSConnectionPool(host='www.noviteroditeli.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.noviteroditeli.bg', port=443): Read timed out. (read timeout=2)"
novosti.hr,"HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='novosti.hr', port=443): Read timed out. (read timeout=2)"
npru.ac.th,"HTTPSConnectionPool(host='npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.npru.ac.th', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
npz.sk,"HTTPSConnectionPool(host='npz.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
nrsr.sk,"HTTPSConnectionPool(host='nrsr.sk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa926cdcd90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nrsr.sk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91d789940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ns.ac.rs,"HTTPSConnectionPool(host='ns.ac.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa925f1c760>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ns.ac.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa925a91370>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nsc.ru,"HTTPSConnectionPool(host='nsc.ru', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa91d789940>, 'Connection to nsc.ru timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nsc.ru', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa91d631640>, 'Connection to nsc.ru timed out. (connect timeout=2)'))"
nsk.hr,"HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nsk.hr', port=443): Read timed out. (read timeout=2)"
nso.go.th,"HTTPSConnectionPool(host='nso.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fa91da199d0>, 'Connection to nso.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='nso.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fa92668c880>, 'Connection to nso.go.th timed out. (connect timeout=2)'))"
nspm.rs,"HTTPSConnectionPool(host='nspm.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa91d76c700>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.nspm.rs', port=80): Read timed out. (read timeout=2)"
nstru.ac.th,"HTTPSConnectionPool(host='nstru.ac.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa91fed42b0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='nstru.ac.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa927342850>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nta.ac.in,"HTTPSConnectionPool(host='nta.ac.in', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='nta.ac.in', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
nta.go.jp,"HTTPSConnectionPool(host='nta.go.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa91ec40c70>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nta.go.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91ec40f40>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nttdocomo.co.jp,"HTTPSConnectionPool(host='nttdocomo.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa91f06c400>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nttdocomo.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91ea9c580>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ntua.gr,"HTTPSConnectionPool(host='ntua.gr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa923b9ec40>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='ntua.gr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa926bd9250>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
nu6i-bg-net.com,"HTTPSConnectionPool(host='nu6i-bg-net.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nu6i-bg-net.com', port=443): Read timed out. (read timeout=2)"
nucleodoconhecimento.com.br,"HTTPSConnectionPool(host='nucleodoconhecimento.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nucleodoconhecimento.com.br', port=443): Read timed out. (read timeout=2)"
number1.co.il,"HTTPSConnectionPool(host='number1.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='number1.co.il', port=80): Read timed out. (read timeout=2)"
nusalist.com,"HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nearbymalaysia.com', port=443): Read timed out. (read timeout=2)"
nuv.cz,"HTTPSConnectionPool(host='nuv.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='www.nuv.cz', port=80): Read timed out. (read timeout=2)"
nwu.ac.za,"HTTPSConnectionPool(host='nwu.ac.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa926ca5be0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nwu.ac.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91fd7a8e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyf.hu,"HTTPSConnectionPool(host='nyf.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa921b38a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='nyf.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa922146520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
nyiregyhaza.hu,"HTTPSConnectionPool(host='www.nyiregyhaza.hu', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.nyiregyhaza.hu', port=443): Read timed out. (read timeout=2)"
nzarn.org.nz,"HTTPSConnectionPool(host='nzarn.org.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='nzarn.org.nz', port=443): Read timed out. (read timeout=2)"
nzlii.org,"HTTPSConnectionPool(host='nzlii.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='nzlii.org', port=80): Read timed out. (read timeout=2)"
oae.go.th,"HTTPSConnectionPool(host='oae.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa92632e4c0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='oae.go.th', port=80): Read timed out. (read timeout=2)"
oamk.fi,"HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oamk.fi', port=443): Read timed out. (read timeout=2)"
obraspublicas.gob.ec,"HTTPSConnectionPool(host='obraspublicas.gob.ec', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'obraspublicas.gob.ec' doesn't match either of 'www.agricultura.gob.ec', 'agricultura.gob.ec'"")))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
obz.hr,"HTTPSConnectionPool(host='obz.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa920a38610>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='obz.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91ed77310>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oca.com.uy,"HTTPSConnectionPool(host='oca.com.uy', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa925475580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='oca.com.uy', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fa91fd9a0d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
